draft-ietf-httpbis-priority.md

```diff
@@ -424,24 +429,32 @@ initial priority of a response, or to reprioritize a response or push stream. It
 carries the stream ID of the response and the priority in ASCII text, using the
```

Comment:
IIUC, this does not use HPACK then? Maybe worth it to point that out explicitly.

Reply:
Only certain frames can drive the HPACK state by HTTP/2 definition. I think mentioning HPACK here would just be confusing.